src/create.js

```diff
@@ -34,16 +35,27 @@ const create = (options = {}, user = {}, serviceToken = '', caseReference = '',
     logger.info('Default Site ID is being used.');
   }
 
-  const uri = `${options.apiBaseUrl}/users/${user.id}/payments`;
+  const uri = `${options.apiBaseUrl}/card-payments`;
+
   const body = {
     amount,
-    reference: `${options.serviceIdentification}$$$${caseReference}$$$${siteId}$$$${feeCode}`,
+    ccd_case_number: `${caseReference}`,
```

Comment:
can just be: `ccd_case_number: caseReference,`

Reply:
Done